docs/get-started/_index.md

```diff
@@ -7,119 +7,7 @@ layout: "empty"
 empty_node: true
```

Comment:
This is actually not an empty node anymore and it should also not have the `layout: "empty"`
```suggestion
```

Reply:
Ah okay I commented out both of those lines temporarily to test it locally but figured you'd had these lines in here for a reason so left as-is! I'll change them